docs/algorand/for-end-users/obtaining-test-algo.mdx

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+---
+import myAlgoWallet from "@site/static/img/myalgowallet.png";
+import mnemonicphrase from "@site/static/img/mnemonicphrase.png";
+import algoDispenser from "@site/static/img/algorand-dispenser.png";
+import algoWalletDashboard from "@site/static/img/algorand-wallet-dashboard.png";
+
+# Configuring Algorand Wallet and obtaining Test ALGO
+<p style={{textAlign:"justify"}}>
+In order to interact with the **Milkomeda Algorand Rollup bridge**, you need to configure native Algorand Wallet. We recommend using MyAlgo wallet to setup a testnet wallet compatible with the Milkomeda A1 bridge (required to move funds between Algorand testnet and the Rollup).
+
+<h4>Configuring myAlgo Wallet</h4>
+
+
+First, go to https://wallet.myalgo.com/new-account and after accepting the _Terms of Service_ click _Continue_ to start configuring your wallet. In the next step you will be asked to create a password. Then you should **create a new randomly generated account**.
+
+<img src={myAlgoWallet} className="img-full" alt="" />
+
+In the next step, you will be asked to save the Mnemonic Phrase.
+
+<img src={mnemonicphrase} className="img-full" alt="" />
+
+:::info
+Remember to take note of the following Mnemonic Phrase words in the correct order. They make the recovery phrase for your new account. 
+:::
+:::caution
+Do not share these words with anyone! These words can be used to steal all your funds.
+:::
+Once Mnemonic Phrase has been successfully verified, your MyAlgo wallet has been configured and all that's left is to name your wallet with any name of your choice.
+
+<h4>Obtaining Test ALGO</h4>
+
+To interact with Bridge dApp you need to have ALGO tokens (tALGO tokens on devnet). 
```

Comment:
```suggestion
To interact with the Bridge dApp you need to have ALGO tokens (tALGO tokens on devnet). 
```

Reply:
They're not actually called "tALGO" on the testnet they're just called ALGO

Reply:
Also, in these instructions you never told the user to switch the wallet to the testnet.